bedrock/mozorg/templates/mozorg/rise25/rise25.html

```diff
@@ -0,0 +1,169 @@
+
+{% block page_desc %}In honor of our 25th anniversary we’re recognizing 25 rising stars. These are the people we see shaping the future of
+the internet for the better over the next quarter century and beyond. Submit your nominations today.{% endblock %}
+
+{% block site_header %}
+  {% with hide_nav_cta=True %}
+    {% include 'includes/protocol/navigation/navigation.html' %}
+  {% endwith %}
+{% endblock %}
+
+
+{% block page_css %}
+  {{ css_bundle('protocol-split') }}
+  {{ css_bundle('rise-25') }}
+{% endblock %}
+
+
+{% set nomination_cards = [
+  {
+    "title": "Activists",
+    "desc": "We’re recognizing the activists using the internet to drive social and political change. Nominate individuals using the
+    internet to amplify their voices and make a difference in the world.",
+    "href": "https://mozilla.formstack.com/forms/activists",
+  },
+  {
+    "title": "Builders",
+    "desc": "These are the engineers and technical people building the infrastructure of the internet. Nominate the builders shaping
+    the technical side of the web, making it faster, more secure and accessible to everyone.",
+    "href": "https://mozilla.formstack.com/forms/builders",
+  },
+  {
+    "title": "Artists",
+    "desc": "These are the creative forces creating innovative and thought-provoking digital artwork. Nominate artists who use the
+    internet as their canvas to inspire others and rethink what's possible online.",
+    "href": "https://mozilla.formstack.com/forms/artists",
+  },
+  {
+    "title": "Creators",
+    "desc": "These are the content creators using storytelling to build community online. Nominate the filmmakers, educators,
+    comedians and social media creators inspiring their audiences and sparking important conversations.",
+    "href": "https://mozilla.formstack.com/forms/creators",
+  },
+  {
+    "title": "Advocates",
+    "desc": "This category is for the people shaping the policies and regulations governing the internet. Nominate the policymakers,
+    lawyers and advocates who are fighting for an open, free internet.",
+    "href": "https://mozilla.formstack.com/forms/advocates",
+  }
+]%}
+
+{% block content %}
+  <section class="rise25-hero">
+      <div class="hero-static-container">
+        <div class="hero-text-container mzp-l-content">
+            <img src="/media/img/mozorg/rise25/RISE25.svg" width="171" height="35">
+            <h1>In search of<br/> the visionaries<br/> and builders of<br/> tomorrow</h1>
+            <div class="play-container">
+              <span class="play-button">
+                <svg width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path d="M23.3047 15.0008L10.8493 22.1919L10.8493 7.80973L23.3047 15.0008Z" fill="white" />
+                  <circle cx="15" cy="15" r="14.5908" stroke="white" stroke-width="0.818462" />
+                </svg>
+              </span>
+              <p>Play video</p>
```

Comment:
Issue (potentially blocking): This `<div>` isn't accessible (via keyboard or screen reader). Might it be possible to use a `<button>` element instead maybe?

Reply:
Question (non-blocking): do we want any GA events around the video play interaction?